### ISSUE TITLE
Decline edit summary improvements

### ISSUE THREAD

**Issue by perryprog** [WPAFC/afch-rewrite#126]
(Related: #103)

Currently the built edit summary for a decline is simply made up from content of the options in `select#declineReason` for the decline template. This can lead to oddly worded edit summaries like

> Declining submission: v - Submission is improperly sourced and nn - Submission is about a topic not yet shown to meet general notability guidelines (be more specific if possible)

Ideally, this would be worded closer to something like
> Declining submission: improperly sourced and topic is not yet shown to meet the [[WP:GNG|general notability guidelines]].

While this is a pretty darn minor issue, I think it's worth being addressed. One way to go about this might be to extract the list of decline reasons to a JS object (or JSON—using JSON5 here for demonstration purposes) which could look something like:
```json5
{
  // [...]
  groups: [
    {
      label: "Notability",
      options: [
        {
          value: "neo",
          reviewerText: "neo – submission is about a neologisim not yet shown to meet notability guidelines",
          editSummaryText: "neologisim not yet shown to meet notability guidelines"
        },
      ]
    }
  ]
}
```
I'm not totally in love with this because it's ridiculously over-verbose, so there's probably a better solution, ideally one that doesn't violate DRY as much as the proposed JSON(5) would.

<sub>(Also, open en dashes—not em dashes, which I normally favor—look much better in both the list and edit summary ;).)</sub>

<details>
<summary>Referenced code</summary>

https://github.com/WPAFC/afch-rewrite/blob/9002cd4e91bb37b4e4bf62bec4858736015c986b/src/modules/submissions.js#L2271-L2296
https://github.com/WPAFC/afch-rewrite/blob/9002cd4e91bb37b4e4bf62bec4858736015c986b/src/templates/tpl-submissions.html#L201-L249
</details>

**Comment by perryprog** [comment]
One other minor thing could be an additional note in the edit summary when a reviewer comment is added on top of the decline reasons.

**Comment by primefac** [comment]
I don't think if (and what) the reviewer commented is particularly necessary on (what is supposed to be) a relatively short edit summary.

**Comment by perryprog** [comment]
Fair enough—I mainly just dislike how you currently can't tell if there was a comment included in a deny without checking the diff, but it's a pretty insignificant thing.

**Comment by primefac** [comment]
Not saying it can't be done (even just putting _+comment_ might work), just giving my thoughts.

**Comment by perryprog** [comment]
Ah, my apologies, I had misread your message. Yeah, a "+ comment" is more what I was thinking; sorry for the confusion.

**Comment by helper-uttam** [comment]
Hi perryprog, If I am not wrong then this issue is somehow related to grammatical error?

I mean that the mentioned sentence " Declining submission: v - Submission is improperly sourced and nn - Submission is about a topic not yet shown to meet general notability guidelines (be more specific if possible) " has to be written in more efficient way? 
Please let me know about this.
